### ISSUE TITLE
Annotation accepting multiple values fail to bind for Kotlin classes

### ISSUE THREAD

**Issue by winterDroid** [JakeWharton/butterknife#1539]
In order to migrate to AndroidX I just wanted to upgrade Butterknife from 8.8.1 to 9.0.0. Unfortunately if fails on runtime on views can't be found.

It seems all annotations accepting multiple IDs at once (e.g. `@BindViews`, `@OnClick`) used in Kotlin classes are not working anymore. They'll be resolved to something like `Utils.findRequiredView(source, 2131493421, "field 'sample'")` where a constant instead of a reference to `R` is used.

As a workaround I need to create for each referenced ID an actual `@BindView` view binding. Then the IDs are correctly resolved. 

After digging into the history and code, I assume that this issue started after #1244

**Comment by winterDroid** [comment]
Duplicate of #1523
